### PR TITLE
tutorials: sandboxfy grafana fundamentals

### PR DESCRIPTION
**Pull request by tonypowa**
## what's this PR about?

Recreates the existing [tutorial](https://grafana.com/tutorials/grafana-fundamentals/) by adding the interactive elements, and running the transformer tool , so future changes to this tutorial will be picked up by the Git workflow in the grafana/killercoda repo , which removes the need for manual edits in the environment.

See parallel PR in Killercoda's side: https://github.com/grafana/killercoda/pull/102

### CODE REVIEW

File: docs/sources/tutorials/grafana-fundamentals/index.md [hunk 1]
```diff
@@ -42,11 +49,25 @@ Alternatively, you can also watch our Grafana for Beginners series where we disc
 - [Docker Compose](https://docs.docker.com/compose/) (included in Docker for Desktop for macOS and Windows)
 - [Git](https://git-scm.com/)
 
-### KillerCoda sandbox environment (Alternative)
+{{% /class %}}
```
Comment (Jayclifford345) [inline docs/sources/tutorials/grafana-fundamentals/index.md:52]: What does this do? You will need to wrap this one in an ignore as the transformer won't know how to handle this yet

File: docs/sources/tutorials/grafana-fundamentals/index.md [hunk 2]
```diff
@@ -42,11 +49,25 @@ Alternatively, you can also watch our Grafana for Beginners series where we disc
...
 
-If you would prefer to follow along with this tutorial without needing to set up a local environment, you can use the [KillerCoda sandbox environment](https://killercoda.com/grafana-labs/course/full-stack/tutorial-enviroment).
+<!-- INTERACTIVE ignore START -->
 
-{{% /class %}}
+{{< admonition type="tip" >}}
+Alternatively, you can try out this example in our interactive learning environment: [Get started with Grafana Alerting](https://killercoda.com/grafana-labs/course/full-stack/tutorial-enviroment).
```
Comment (Jayclifford345) [inline docs/sources/tutorials/grafana-fundamentals/index.md:57]: I think the url needs to be changed to match where you have placed the tutorial:
```
grafana/grafana-fundamentals
```
also the description

File: docs/sources/tutorials/grafana-fundamentals/index.md [hunk 3]
```diff
@@ -405,18 +463,35 @@ Check out our [advanced alerting tutorial](http://grafana.com/tutorials/alerting
 
 {{< /docs/ignore >}}
```
Comment (Jayclifford345) [inline docs/sources/tutorials/grafana-fundamentals/index.md:464]: Just wanted to check to see if this was a `{{< /docs/ignore >}}` without a starting tag.

I also noticed a few short codes the transformer won't support yet :
```
{{< youtube id=“ClLp-iSoaSY” >}}
{{< admonition type=“note” >}}
```
If we can wrap these in an interactive ignore that would be awsome!

File: docs/sources/tutorials/grafana-fundamentals/index.md [hunk 4]
```diff
@@ -405,18 +463,35 @@ Check out our [advanced alerting tutorial](http://grafana.com/tutorials/alerting
...
+
+In this tutorial you learned about fundamental features of Grafana. 
+
+{{< /docs/ignore >}}
+
+<!-- INTERACTIVE ignore START -->
 
 ### KillerCoda sandbox environment (completed tutorial)
 
 Do you want to see the finished result? Check out our [completed KillerCoda sandbox environment](https://killercoda.com/grafana-labs/course/full-stack/tutorial-enviroment-completed) containing the entire demo with dashboards, checks, and data sources configured.
```
Comment (Jayclifford345) [inline docs/sources/tutorials/grafana-fundamentals/index.md:491]: I would just delete this section as it makes sense for us to EOL those old sandboxes to make way for these

**Review by Jayclifford345 — COMMENTED** [review]
Looking awsome my dude! just a couple of questions and edits in there but shouldn't be huge amounts of work I hope :)

File: docs/sources/tutorials/grafana-fundamentals/index.md (hunk 1 above)
Comment (tonypowa) [reply docs/sources/tutorials/grafana-fundamentals/index.md:52]: that puts the text into a grey box. it was already there
i will remove it 👍🏼

File: docs/sources/tutorials/grafana-fundamentals/index.md (hunk 2 above)
Comment (tonypowa) [reply docs/sources/tutorials/grafana-fundamentals/index.md:57]: done!
which description ?

Comment (Jayclifford345) [reply docs/sources/tutorials/grafana-fundamentals/index.md]: it currently says
`Get started with Grafana Alerting` wondered if that was correct

**Review by Jayclifford345 — APPROVED** [review]
LGTM

**Review by irenerl24 — COMMENTED** [review]
Made some minor edits, other than that, LGTM! Thank you!

**Pull request merged**